src/logger.js

```diff
@@ -19,10 +19,15 @@ const logger = new (winston.Logger)({
  * @param {*} customMessage 
  */
 export const logError = (error, customMessage = null, isStack = true) => {
-  logger.error(JSON.stringify({ type: 'Error',
-    message: error.message,
-    stack: isStack ? error.stack : null,
-    customMessage }));
+  if (getEnv('OUTPUT_TYPE') === 'terminal') {
```

Comment:
Lets make it a switch case to be able to easier add more output modes :)

Reply:
This is better for this use case and we don't have plan to other format, are we?